cmd/bootstrap/transit/main.go

```diff
@@ -65,6 +66,12 @@ func main() {
 	flag.StringVar(&flowBucket, "flow-bucket", "flow-genesis-bootstrap", "Storage for the transit server")
 	flag.Parse()
 
+	// always print version information
```

Comment:
Operators generally provide us with all the output they got when they ran the transit script and something failed. To avoid going back and checking with them if they have the right version of the script, I thought it would be best to just print the version our for each run so that the output they provide will always include it.